WordPress/Classes/Stores/StatsPeriodStore.swift

```diff
@@ -170,7 +183,45 @@ private extension StatsPeriodStore {
 
     func setAllAsFetching() {
         state.fetchingPostsAndPages = true
+        state.fetchingSearchTerms = true
+    }
+
+    /// This method modifies the 'Unknown search terms' row and changes its location in the array.
+    /// - Find the 'Unknown search terms' row
+    /// - Change the label
+    /// - Remove the row from the array
+    /// - Insert the row at the beginning of the array
+    /// NOTE: When the backend is updated, maybe it will return the unknown row at the top
+    /// of the array, making this unnecessary.
+    ///
+    func reorderSearchTerms(_ searchTerms: StatsGroup?) -> [StatsItem]? {
```

Comment:
Hey Stephnie 👋 Are we recording the search terms because the order they come back from the API doesn't fit design? 

I wonder if it would make more sense to name this method more specifically for "unknown search terms" as it deals with a specific label/row. Or maybe make the reordering item in an array and make that generic or an Array extension?
Something like:
`func rearrange<T>(array: Array<T>, fromIndex: Int, toIndex: Int) -> Array<T>{
    var arr = array
    let element = arr.remove(at: fromIndex)
    arr.insert(element, at: toIndex)

    return arr
}`

Reply:
Hey @yaelirub .

Thanks for the feedback.

The problem is not the API itself. After the results are received, `WPStatsServiceRemote:operationForSearchTermsForDate` manually adds the `Unknown Search Terms` row to the end of the array. @jklausa and I have discussed this as he's refactoring the backend. 
I don't know what the final version of the backend will look like, but I honestly expect this method to go away and, if necessary, another method to handle what is provided.